var/spack/repos/builtin/packages/babelstream/package.py

```diff
@@ -45,7 +56,30 @@ class Babelstream(CMakePackage, CudaPackage, ROCmPackage):
     # Some models need to have the programming model abstraction downloaded -
     # this variant enables a path to be provided.
     variant("dir", values=str, default="none", description="Enable Directory support")
-
+    variant("submodel", values=str, default="none", description="SYCL2020 -> choose between usm and acc methods")
```

Comment:
```suggestion
    variant("submodel", values=("usm", "acc"), when="+sycl2020", default="usm", description="SYCL2020 -> choose between usm and acc methods")
```

Reply:
Then we can delete the conflicts statement below that implements this when clause.